package.json

```diff
@@ -20,7 +20,10 @@
     "install:docs": "yarn --cwd website install",
     "start:docs": "yarn --cwd website start",
     "build:docs": "yarn --cwd website build",
-    "format:docs": "prettier --write docs"
+    "format:docs": "prettier --write docs",
+    "prepare": "yarn build",
+    "changeset": "changeset",
+    "version": "yarn changeset version && yarn install --frozen-lockfile"
```

Comment:
I forget the most recent reference, but isn't the `yarn install` _meant_ to update the lockfile due to changes after `changeset version` (and thus don't freeze it).

Reply:
It's meant to fail if the lockfile changes during install which is our typical command we use in our version task.

Reply:
See, e.g.: https://github.com/FormidableLabs/victory/blob/main/package.json#L118

```js
"version": "pnpm changeset version && pnpm install --no-frozen-lockfile",
```

https://github.com/FormidableLabs/victory/pull/2398